src/Qir/Runtime/stdlib/include/qir_stdlib.h

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <cstdint>
+
+#ifdef _WIN32
+#define QIR_SHARED_API __declspec(dllexport)
+#else
+#define QIR_SHARED_API
+#endif
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    struct QirString;
+    struct QirArray;
+    struct QirCallable;
+    struct QirTuple;
+    struct QirBigInt;
+
+    enum PauliId : int8_t
```

Comment:
Possibly redundant enum definition?

Reply:
Looks like it's not redundant, as the enum values must be declared somewhere for C to use them.